deno.json

```diff
@@ -83,9 +83,6 @@
 
     "@fresh/plugin-tailwind": "jsr:@fresh/plugin-tailwind@^0.0.1-alpha.7",
     "@std/front-matter": "jsr:@std/front-matter@^1.0.5",
-    "fresh/compat": "./src/compat/mod.ts",
-    "fresh/dev": "./src/dev/mod.ts",
-    "fresh/runtime": "./src/runtime/shared.ts",
```

Comment:
These imports are redundant because of the `fresh` import that's already defined.